vendor/vendor-android/src/main/kotlin/com/malinskiy/marathon/android/RemoteFileManager.kt

```diff
@@ -44,7 +44,7 @@ class RemoteFileManager(private val device: AndroidDevice) {
     }
 
     companion object {
-        const val MAX_FILENAME = 255
+        const val MAX_FILENAME = 254 //we need 1 more char for fileNumber in case of using video attachment > 180 && apiLevel < 34
```

Comment:
This constant is not specific for video files so it should never change. It's a detail about Android filesystem implementation. I suggest changing the logic of the code, not the constants here